tests/components/konnected/test_panel.py

```diff
@@ -92,9 +93,6 @@ def mock_constructor(host, port, websession):
         options=device_options,
```

Comment:
Instead of creating the data and options first and using that to create a mock entry, just pass the configuration to `async_setup_component`. It will set up the integration and use the config yaml schemas of the integration before importing the config into a config entry.